repository.datadog.yml

```diff
@@ -0,0 +1,7 @@
+---
+schema-version: v1
+kind: mergequeue
+team: ci-interfaces
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** I don't see any doc for the `team` option. ‏Does it correspond do the GitHub team like `DataDog/[team]`? Shouldn't the team be `rum-browser`?

Reply:
good question ... I don't think we use the field for the moment. Let's change to rum-browser